index.html

```diff
@@ -0,0 +1,229 @@
+        <div class="brand">
+          <img src="./img/logo_svg.png" alt="logo" height="26px" width="26px">
+          <h1 style="padding-left: 5px;">Dev</h1>
+        </div>
+        <ul class="navbar_buttons">
+          <li><a href="#about_me" class="navbutton">About me</a></li>
+          <li><a href="#skills" class="navbutton">Skills</a></li>
+          <li><a href="#portfolio" class="navbutton">Portfolio</a></li>
+          <li><a href="#contact_us" class="navbutton">Contact</a></li>
+        </ul>      
+      </nav>
+      <div class="container">
+        <div class="cover_photo">
+          <img src="./img/cover_layer.png" class="layer" alt="layer">
+          <h1>Welcome!</h1>
+          <h2>My name is Konrad.</h2> 
+          <div class="social_media">
+            <ul class="social_media">
+              <li>
+                <a href="https://twitter.com/" target="_blank">
+                  <span class="fa-stack fa-lg">
+                    <i class="fa fa-circle fa-stack-2x"></i>
+                    <i class="fab fa-twitter fa-stack-1x"></i>
+                  </span>
+                </a>
+              </li>
+              <li>
+                <a href="https://www.linkedin.com/in/konrad-kalita-72121b182/" target="_blank">
+                  <span class="fa-stack fa-lg">
+                    <i class="fa fa-circle fa-stack-2x"></i>
+                    <i class="fab fa-linkedin-in fa-stack-1x"></i>
+                  </span>
+                </a>
+              </li>
+              <li>
+                <a href="https://github.com/" target="_blank">
+                  <span class="fa-stack fa-lg">
+                    <i class="fa fa-circle fa-stack-2x"></i>
+                    <i class="fab fa-github fa-stack-1x"></i>
+                  </span>
+                </a>
+              </li>
+              <li>
+                <a href="https://facebook.com/" target="_blank">
+                  <span class="fa-stack fa-lg">
+                    <i class="fa fa-circle fa-stack-2x"></i>
+                    <i class="fab fa-facebook-f fa-stack-1x"></i>
+                  </span>
+                </a>
+              </li>
+            </ul>
+          </div>   
+        </div>
+      </div>
+    </header>
+    
+    <section class="contact_and_cv">
+      <h2>I am happy to help you with any problem!</h2>
+      <div class="button_container">
+        <button type="button" class="button" onclick="alert('Contact redirection')">Contact me!</button>
+        <button type="button" class="button" onclick="alert('CV redirection')">Check my CV</button>
+      </div>
+    </section>
+
+    <section class="about_me" id="about_me">
+      <h2>About me</h2>
+      <div class="photo_personalData_description">
+        <img src="./img/avatar.png" alt="Photo of me" width="96px" height="96px">
+        <div class="personalData">
+          <ul>
+            <li>Full name: <b class="about_me_distinctive_text">Konrad Kalita</b></li>
```

Comment:
plus za użycie listy 😃 polecam również zapoznać się z tagami `description list` https://developer.mozilla.org/pl/docs/Web/HTML/Element/dl 
Mało popularne, natomiast jeśli chodzi o semantykę to jego zastosowanie byłoby tutaj jak najbardziej poprawne :)